ga4gh/backend.py

```diff
@@ -700,6 +710,38 @@ def featuresGenerator(self, request):
             request.page_token, request.page_size,
             request.feature_types, parentId, request.name, request.gene_symbol)
 
+    def phenotypesGenerator(self, request):
+        """
+        Returns a generator over the (phenotypes, nextPageToken) pairs
+        defined by the (JSON string) request
+        """
+        # TODO make paging work using SPARQL?
```

Comment:
for both `phenotypesGenerator` and `genotypesPhenotypesGenerator` it seems like paging is not implemented efficiently... the entire set of results is requested on each page request, unless I'm missing something (`request.page_token` and `request.page_size` don't appear to be utilized anywhere in the stack)